project_admin/tests/tests.py

```diff
@@ -43,3 +43,174 @@ def test_admin_login_fail(self):
         response = c.post("/project-admin/login/", {'password': 'meep'})
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, "Password incorrect")
+
+    def test_config_homepage_text_logged_out(self):
+        """
+        Test config_homepage_text function when logged out
+        """
+        c = Client()
+        response = c.get("/project-admin/config-homepage-text/")
+        self.assertEqual(response.status_code, 302)
+
+    def test_get_config_homepage_text(self):
+        """
+        Test making a get request to
+        config_homepage_text function when logged in.
+        """
+        c = Client()
+        response = c.post("/project-admin/login/",
+                          {'password': 'test1234'},
+                          follow=True)
+        response = c.get("/project-admin/config-homepage-text/")
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response,
+                                'project_admin/config-homepage-text.html')
+
+    def test_post_config_homepage_text(self):
+        """
+        Test making a post request to
+        config_homepage_text function when logged in.
+        """
+        c = Client()
+        response = c.post("/project-admin/login/",
+                          {'password': 'test1234'},
+                          follow=True)
+        response = c.post("/project-admin/config-homepage-text/",
+                          {'faq': 'foo',
+                           'about': 'bar',
+                           'overview': 'foo-bar',
+                           'homepage_text': 'foobar!',
+                           'upload_description': 'foo_bar'},
+                          follow=True)
+        self.assertEqual(response.context['config'].faq, 'foo')
+        self.assertEqual(response.context['config'].about, 'bar')
+        self.assertEqual(response.context['config'].overview, 'foo-bar')
+        self.assertEqual(response.context['config'].homepage_text, 'foobar!')
+        self.assertEqual(response.context['config'].upload_description,
+                         'foo_bar')
+
+    def test_config_file_settings_logged_out(self):
+        """
+        Test config_file_settings function when logged out
+        """
+        c = Client()
+        response = c.get("/project-admin/config-file-settings/")
+        self.assertEqual(response.status_code, 302)
```

Comment:
Should also use `self.assertRedirects` as it not only tests the status code but also the target.